bigquery/snippets/dataset/bigquery_create_dataset.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dataset
+
+// [START bigquery_create_dataset]
+
+import (
+	"context"
+
+	"cloud.google.com/go/bigquery"
+)
+
+func createDataset(client *bigquery.Client, datasetID string) error {
```

Comment:
Optionally add function comments to describe what exactly the function does. See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#comment-functions-and-packages.

Reply:
Omitting the function comments given the naming of the files and function names are fairly explicit, but I added a package comment to the test file.